<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeaderbordsWindow</class>
 <widget class="QDialog" name="LeaderbordsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>411</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Leaderboards</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_show">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GoBack">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>630</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Go back to start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Save">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>630</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/recources/Bilder till projekt/thJ20O2MMZ.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/recources/Bilder till projekt/5511133955_de627d006a_z.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>205</width>
     <height>243</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Show leaderboards</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_show_allSByPM">
      <property name="text">
       <string>All season by penalty min</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_show_allSByPS">
      <property name="text">
       <string>All season by penalty shots</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_show_allSByG">
      <property name="text">
       <string>All season by goals</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_show_seasAll">
      <property name="text">
       <string>All leaderboards for the season</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Show for this month:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit_SHow">
        <property name="displayFormat">
         <string>MMM</string>
        </property>
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_show_mByPM">
      <property name="text">
       <string>Month by penalty min</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_show_mByPS">
      <property name="text">
       <string>Month by penalty shots</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_show_mByG">
      <property name="text">
       <string>Month by goals</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_show_mAll">
      <property name="text">
       <string>All leaderbords for the month</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>267</width>
     <height>243</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save leaderboards to file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_save_allSByPM">
      <property name="text">
       <string>All season by penalty min</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_save_allSByPS">
      <property name="text">
       <string>All season by penalty shots</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_save_allSByG">
      <property name="text">
       <string>All season by goals</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_save_allS">
      <property name="text">
       <string>All leaderboards for the season</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Save leaderbords for this month:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit_save">
        <property name="displayFormat">
         <string>MMM</string>
        </property>
        <property name="calendarPopup">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_save_mByPM">
      <property name="text">
       <string>Month by penalty min</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_save_mByPS">
      <property name="text">
       <string>Month by penalty shots</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_save_mByG">
      <property name="text">
       <string>Month by goals</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_save_mAll">
      <property name="text">
       <string>All leaderbords for the month</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
